components/Snackbar.vue

```diff
@@ -76,16 +79,22 @@ export default class Snackbar extends Vue {
   @ModelSync('open', 'toggle', { type: Boolean, default: false })
   isOpen!: boolean
 
+  @Watch('isOpen')
+  timeOut() {
+    if (this.timeout) {
+      setTimeout(() => {
+        this.close()
+      }, this.timeout)
+    }
+  }
```

Comment:
Do we need to clear this timeout if the user explicitly closed this? Would it cause error if `this.close()` is called when the snackbar is closed manually first?